src/main/java/mil/dds/anet/config/SecurityConfig.java

```diff
@@ -106,11 +106,30 @@ public SecurityFilterChain soapSecurityFilterChain(HttpSecurity http) throws Exc
 
   @Bean
   @Order(20)
+  public SecurityFilterChain graphQLWebServicveFilterChain(HttpSecurity http) throws Exception {
+    http.securityMatcher("/graphqlWebService/**") // Apply only to GraphQL endpoints
+        .authorizeHttpRequests(auth -> auth.anyRequest().permitAll());
+
+    // Stateless, disable session management
+    http.sessionManagement(
+        session -> session.sessionCreationPolicy(SessionCreationPolicy.STATELESS));
+    // Disable CSRF
+    http.csrf(AbstractHttpConfigurer::disable);
+    // Configure CSP
+    http.headers(
+        headers -> headers.contentSecurityPolicy(csp -> csp.policyDirectives(soapCsp.toString())));
+    // Configure Referrer Policy
+    http.headers(header -> header.referrerPolicy(rp -> rp.policy(NO_REFERRER)));
+    return http.build();
+  }
+
+  @Bean
+  @Order(30)
   public SecurityFilterChain defaultSecurityFilterChain(HttpSecurity http) throws Exception {
     http.authorizeHttpRequests(authorize -> authorize
         // These are public
         .requestMatchers(AdminResource.ADMIN_DICTIONARY_RESOURCE_PATH, HomeResource.LOGOUT_PATH,
-            AssetConfig.ASSETS_PATH, AssetConfig.IMAGERY_PATH)
+            AssetConfig.ASSETS_PATH, AssetConfig.IMAGERY_PATH, AssetConfig.GRAPHQL_WEB_SERVICE)
```

Comment:
Remove `, AssetConfig.GRAPHQL_WEB_SERVICE` here, it is handled by a higher-prio security config (above).